configs/ds_pretrain_gpt2.yml

```diff
@@ -0,0 +1,64 @@
+{
+   "num_nodes":"1",
+   "num_gpus":"8",
+   "model-parallel-size":"4",
+   "num-layers":"24",
+   "hidden-size":"1024",
+   "num-attention-heads":"16",
+   "seq-length":"1024",
+   "max-position-embeddings":"1024",
+   "batch-size":"9",
```

Comment:
Why 9?

Reply:
It’s taken from the corresponding examples script